observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemRegistrar.java

```diff
@@ -70,19 +83,21 @@ class MicrometerSubsystemRegistrar implements SubsystemResourceDefinitionRegistr
             "io.micrometer"
     };
 
+    @Deprecated
     public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
```

Comment:
Given the field is deprecated and there is a replacement, should this add `.setDeprecated(ModelVersion.create(2, 0, 0))`?